code/frameworks/angular/src/client/index.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="webpack-env" />
```

Comment:
If we add all these to these packages do we need to add a dependency?

Reply:
sorta maybe? I guess..